jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/ContainerParameters.java

```diff
@@ -266,11 +270,15 @@ public String getFilesModificationTime() {
     if (System.getProperty(PropertyNames.CONTAINER_FILES_MODIFICATION_TIME) != null) {
       return System.getProperty(PropertyNames.CONTAINER_FILES_MODIFICATION_TIME);
     }
-    return filesModificationTime;
+    return filesModificationTime.getOrNull();
```

Comment:
`getOrElse` with the default value of `EPOCH_PLUS_SECOND`, perhaps? Realistically, it should not be null, unless the user sets `null` explicitly, but easier to be safe.

Reply:
This is now handled by `convention("EPOCH_PLUS_SECOND")` and `convention("EPOCH")` on the (now final) Property objects